docs/Styling.md

```diff
@@ -1,22 +1,26 @@
 # Styling Ignite apps
 
-Ignite's approach to styling is, like many other things in Ignite, straightforward and direct.
+[Back to README](./README.md)
+
+Ignite's approach to styling individual components is, like many other things in Ignite, straightforward and direct.
+
+If you're looking to set app-wide styles such ass, check out the [Theming](./Theming.md) documentation.
```

Comment:
two comments here:
1. `such as` not `ass`
2. such as...? or should the sentence not have `such as`?